pymc/pytensorf.py

```diff
@@ -92,6 +92,22 @@
 ]
 
 
+# Temporarily add an Aesara dispatch for PyTensor TensorVariable to inform users
+# of incorrect mixing of libraries
+try:
+    from aesara.tensor import _as_tensor_variable as _as_tensor_variable_aesara
+
+    @_as_tensor_variable_aesara.register(TensorVariable)
+    def raise_informative_error(*args, **kwargs):
+        raise TypeError(
+            "You are calling an Aesara function with PyTensor variables. "
```

Comment:
Should we add more information here, for people who might not have read the announcement?

Reply:
Sure, feel free to add a suggestion as review / push to the branch